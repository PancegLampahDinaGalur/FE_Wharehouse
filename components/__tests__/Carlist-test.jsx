import * as React from "react";
import renderer from "react-test-renderer";

import Carlist from "../CarList";

it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <Carlist
        image={require("@/assets/images/killua.png")}
        carName={"Inova"}
        passengers={5}
        baggage={4}
        price={"5000"}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
