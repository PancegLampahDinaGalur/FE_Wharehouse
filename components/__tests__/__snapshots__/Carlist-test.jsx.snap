// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#fff",
      "borderColor": "rgba(0,0,0,0.2)",
      "borderRadius": 2,
      "borderWidth": 0.5,
      "elevation": 2,
      "marginBottom": 20,
      "padding": 20,
      "shadowColor": "rgba(0,0,0,1)",
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "gap": 20,
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "column",
        }
      }
    >
      <Image
        source={1}
        style={
          {
            "height": 80,
            "objectFit": "contain",
            "width": 80,
          }
        }
      />
    </View>
    <View
      style={
        {
          "flexDirection": "column",
        }
      }
    >
      <Text
        style={
          {
            "fontSize": 14,
          }
        }
      >
        Inova
      </Text>
      <View
        style={
          {
            "alignItems": "baseline",
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "flexDirection": "row",
              "gap": 2,
            }
          }
        >
          <Text />
          <Text
            style={
              {
                "color": "#8A8A8A",
              }
            }
          >
            5
          </Text>
        </View>
        <View
          style={
            {
              "alignItems": "center",
              "flexDirection": "row",
              "gap": 2,
            }
          }
        >
          <Text />
          <Text
            style={
              {
                "color": "#8A8A8A",
              }
            }
          >
            4
          </Text>
        </View>
      </View>
      <Text
        style={
          {
            "color": "#5CB85F",
          }
        }
      >
        RpÂ 5.000,00
      </Text>
    </View>
  </View>
</View>
`;
